apps/src/javalab/NameFileDialog.jsx

```diff
@@ -24,6 +24,20 @@ export default class NameFileDialog extends Component {
     };
   }
 
+  componentDidUpdate(prevProps) {
+    if (!prevProps.isOpen && this.props.isOpen) {
+      this.textInput.setSelectionRange(0, 0);
+    }
+  }
+
+  onKeyDown = event => {
+    if (event.key === 'Enter') {
```

Comment:
nit: I think we have `KeyCodes` constants, should we use that here to be safe?

Reply:
oh nice, did not know that existed! I do see some other examples where we don't use `KeyCodes` for similar situations. The Drone gods were kind and gave me a passing build on the first try for once, so I'm just going to roll with it this time.